oem/ibm/pfw-sms-utils/try_pfw_sms_ibm.cpp

```diff
@@ -0,0 +1,43 @@
+// unit test stub
```

Comment:
Please remove this file as this can be enabled later with Gtest frame work

Reply:
I will remove this.  And plan to replace with gtest.